trade-finance-manager-ui/server/controllers/utilisation-reports/record-corrections/create-record-correction-request/form-helpers.ts

```diff
@@ -0,0 +1,50 @@
+import { isNonEmptyString, RECORD_CORRECTION_REASON, RecordCorrectionReason } from '@ukef/dtfs2-common';
+import { CreateRecordCorrectionRequestFormValues } from '../../../../types/view-models';
+
+/**
+ * Values for the record correction reasons as a string array.
+ */
+const RECORD_CORRECTION_REASON_VALUES = Object.values(RECORD_CORRECTION_REASON) as string[];
+
+export type CreateRecordCorrectionRequestFormRequestBody = {
+  reasons?: string | string[];
+  additionalInfo?: string;
+};
+
+/**
+ * Checks if the provided reason is a valid RecordCorrectionRequestReason.
+ * @param reason - The reason to validate.
+ * @returns True if the reason is valid, otherwise false.
+ */
+export function isRecordCorrectionReason(reason: string): reason is RecordCorrectionReason {
+  return isNonEmptyString(reason) && RECORD_CORRECTION_REASON_VALUES.includes(reason);
```

Comment:
Seems that just `RECORD_CORRECTION_REASON_VALUES.includes(reason)` would work?

Reply:
That is a good point, thanks for spotting